package.json

```diff
@@ -147,15 +150,17 @@
     "extract-text-webpack-plugin": "^3.0.1",
     "file-loader": "^1.0.0",
     "html-webpack-plugin": "^2.30.1",
-    "husky": "^0.14.3",
+    "husky": "^0.15.0-rc.9",
```

Comment:
Not a huge fan of having rc releases in. Looks like they did 3 rc versions in one day, currently at rc 12.

Reply:
Yea, good call. I'll revert the husky changes.